src/main/java/spoon/support/compiler/ProgressLogger.java

```diff
@@ -0,0 +1,43 @@
+package spoon.support.compiler;
```

Comment:
The license header is missing. Could you add it please, else it will break the CI on master: merge with master and `mvn license:format`